pkgs/top-level/all-packages.nix

```diff
@@ -102,6 +102,10 @@ in
 
   ### BUILD SUPPORT
 
+  auditBlasHook = makeSetupHook
```

Comment:
Should give this a name. (Really, they all should have names, but that's a mass rebuild we've put off.)